packages/icon-builder/test/index.js

```diff
@@ -186,15 +184,14 @@ describe('--mui-require', function() {
       var relativeOptions = _.extend({}, options, { muiRequire: 'relative' });
```

Comment:
Haha, that's what I thought! But I wasn't going to start battle at this point. 

There's lots about this package that is "old-school" (two-year-old school, crazy isn't it?!)